app/controllers/repositories_controller.rb

```diff
@@ -43,15 +43,17 @@ def find_project
   end
 
   def repository_params
-    params.require(:repository).permit(:path, :heroku_buildpacks, :runtime_env_variables, :build_env_variables, :build_type, addon_ids: [], web_processes_attributes: %i[id name command])
+    params.require(:repository).permit(
```

Comment:
I guess you should also change params in `app/controllers/gitlab_integration/repositories_controller.rb`

Reply:
I didn't add it here for now

Reply:
done